.github/workflows/test_frameworks.yml

```diff
@@ -86,9 +94,10 @@ jobs:
             gunicorn eventlet flup bjoern gevent aiohttp-wsgi uvloop; do
             pip install $name || echo "Failed to install $name with $(python -V 2>&1)" 1>&2
           done
+      # DEV: Inject the latest ddtrace on PyPI while we have this running nightly
       - name: Inject ddtrace
         if: needs.needs-run.outputs.outcome == 'success'
-        run: pip install ../ddtrace
+        run: pip install ddtrace
```

Comment:
🤔 latest "dev" is still useful, no? if we want to catch issues before they are released?

Reply:
Yeah I just realized that too 😅 I'm gonna revert this change